### PR TITLE
Update OIDC section of config.apps.sample.php

### PR DESCRIPTION
**Pull request by mmattel**
## Description
Some updates to the OIDC section in config.apps.sample.php

## Related Issue
- Fixes https://github.com/owncloud/docs/pull/3467#discussion_r611905716

## Motivation and Context
Completeness of parameters

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Database schema changes (next release will require increase of minor version instead of patch)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [x] Documentation ticket raised: will be generated when merged 
- [ ] Changelog item, see [TEMPLATE](https://github.com/owncloud/core/blob/master/changelog/TEMPLATE)

@DeepDiver1975 mind to take a look on the OIDC section in general and approve this PR if you are fine with - many thanks

### CODE REVIEW

File: config/config.apps.sample.php [hunk 1]
```diff
@@ -128,20 +128,29 @@
...
- * If `true`, the login page will automatically be redirected to the OpenID
- * Connect Provider, as when the button is pressed. The default is `false`.
- *
+ * If `true`, the ownCloud login page will redirect directly to the Identity Provider
+ * login without requiring the user to click a button. The default is `false`.
+ *
+ * auto-provision::
+ * If auto-provision is setup, an owncloud user will be created after successful login
+ * using openid connect. If auto-provision is not setup, it is expected that the user exists.
+ * This is where usualy an LDAP setup is required additionally
```
Comment (phil-davis) [inline config/config.apps.sample.php:143]:
```suggestion
 * This is where an LDAP setup is usually required
```

File: config/config.apps.sample.php [hunk 2]
```diff
@@ -128,20 +128,29 @@
...
+* 
  * insecure::
  * Boolean value (`true`/`false`), no SSL verification will take place when talking to the
  * IdP - **DO NOT use in production!**
  *
  * loginButtonName::
  * The name as displayed on the login screen which is used to redirect to the IdP.
+ * By default, the OpenID Connect App will add a button on the login page that will
+ * redirect the user to the Identity Provider and allow authentication via OIDC.
+ * This parameter allows to modify the button text.
```
Comment (phil-davis) [inline config/config.apps.sample.php:153]:
```suggestion
 * This parameter allows the button text to be modified.
```

**Review by phil-davis — COMMENTED** [review]
I just reviewed some English. The content is probably correct, but someone other guru should confirm that!

File: config/config.apps.sample.php [hunk 3]
```diff
@@ -128,20 +128,29 @@
...
  * Additional parameters which are sent to the IdP during the auth requests
  *
  * autoRedirectOnLoginPage::
- * If `true`, the login page will automatically be redirected to the OpenID
- * Connect Provider, as when the button is pressed. The default is `false`.
- *
+ * If `true`, the ownCloud login page will redirect directly to the Identity Provider
+ * login without requiring the user to click a button. The default is `false`.
+ *
+ * auto-provision::
```
Comment (DeepDiver1975) [inline config/config.apps.sample.php:140]: auto provision has some additional properties:
https://github.com/owncloud/openidconnect#setup-auto-provisioning-mode

**Review by DeepDiver1975 — CHANGES_REQUESTED** [review]

File: config/config.apps.sample.php (hunk 3 above)
Comment (mmattel) [reply config/config.apps.sample.php:140]: I have incorporated the change suggested into the auto provisioning text and referenced to the auto config example we already have below. I also squashed. Pls recheck if this change is ok

**Review by DeepDiver1975 — APPROVED** [review]

**Review by phil-davis — APPROVED** [review]

**Pull request merged**